ouroboros-consensus-cardano-test/src/Test/Consensus/Cardano/Generators.hs

```diff
@@ -64,6 +74,35 @@ import           Test.Consensus.Cardano.MockCrypto
   Disk
 -------------------------------------------------------------------------------}
 
+
+-- | Orphans
+instance Arbitrary (TxOut (AlonzoEra c)) where
+  arbitrary = undefined
+
+instance Arbitrary (Script (AlonzoEra c)) where
+  arbitrary = undefined
+
+instance Arbitrary (TxBody (AlonzoEra c)) where
+  arbitrary = undefined
+
+instance Arbitrary (PParams (AlonzoEra c)) where
+  arbitrary = undefined
+
+instance Arbitrary (AuxiliaryData (AlonzoEra c)) where
+  arbitrary = undefined
```

Comment:
@EncodePanda you can get these Arbitrary instances from `Test.Cardano.Ledger.Alonzo.Serialisation.Generators`.